src/types/function.ts

```diff
@@ -23,6 +23,7 @@ export interface IFunctionValues {
   original: any;
   given: any;
   documentInventory: DocumentInventory;
+  context: 'resolved' | 'unresolved';
```

Comment:
@P0lip Why using strings rather than a boolean?

Reply:
like `resolved: false | true`?

Reply:
I actually went ahead and included the whole rule - pretty sure it might be useful for us in future, in particular in custom functions.